.idea/.gitignore

```diff
@@ -1,8 +0,0 @@
-# Default ignored files
```

Comment:
Looks like you accidentally picked up a deleted file with this commit. Can you put this back?